etc/terminal-run

```diff
@@ -21,44 +21,83 @@ function error {
   exit 1
 }
 
-if [ -f /usr/bin/lxterminal ];then
+# Check if the current terminal is in /usr/bin - Wallee
+if ! [ -f /usr/bin/$(cat ~/pi-apps/data/settings/Terminal) ];then
```

Comment:
The `~/pi-apps` part should be changed to $DIRECTORY. This variable allows you to rename the pi-apps folder to anything you want - something we did a lot when testing the new gui.

Reply:
Yeah I removed it for testing and I could not figure out how to reimplement it, took me 2 and a half hours to get the line to "work" in general xD